src/pages/home/report/ReportActionsListItemRenderer.js

```diff
@@ -38,10 +41,12 @@ const propTypes = {
 const defaultProps = {
     mostRecentIOUReportActionID: '',
     linkedReportActionID: '',
+    parentReportAction: {},
 };
 
 function ReportActionsListItemRenderer({
     reportAction,
+    parentReportAction,
```

Comment:
This wasn't added to the memoization of the component which lead to a regression https://github.com/Expensify/App/issues/37466